Xamarin.Forms.Core/Stepper.cs

```diff
@@ -32,7 +32,11 @@ public class Stepper : View, IElementConfiguration<Stepper>
 		public static readonly BindableProperty ValueProperty = BindableProperty.Create("Value", typeof(double), typeof(Stepper), 0.0, BindingMode.TwoWay, coerceValue: (bindable, value) =>
 		{
 			var stepper = (Stepper)bindable;
-			return ((double)value).Clamp(stepper.Minimum, stepper.Maximum);
+			var clampedValue = ((double)value).Clamp(stepper.Minimum, stepper.Maximum);
+			var decimalPlaces = stepper.DecimalPlaces;
+			return decimalPlaces >= 0 && decimalPlaces <= 15
```

Comment:
why the `<=15` ?

Reply:
@StephaneDelcroix **double** has 15 decimal digits of precision

Reply:
@StephaneDelcroix do you think it is unused?

Reply:
I think it's better to validate the value of the DecimalPlacesProperty

Reply:
Makes sense